ci-operator/step-registry/redhat-chaos/application-outages/console/redhat-chaos-application-outages-console-commands.sh

```diff
@@ -22,6 +18,15 @@ echo "Using the flattened version of kubeconfig"
 oc config view --flatten > /tmp/config
 export KUBECONFIG=/tmp/config
 
+OPENSHIFT_PASSWORD="$(cat "$KUBEADMIN_PASSWORD_FILE")"
+console_url=$(oc get routes -n openshift-console console -o jsonpath='{.spec.host}')
+export HEALTH_CHECK_URL="https://$console_url"
```

Comment:
We might also want to add prometheus url check. Thoughts?

Reply:
right now our krkn-hub is only set for one URL, I want to see if i can get this one working before adding another. thoughts?

Reply:
Sounds good!